lessons/06-apis-debugging/INCLASS.md

```diff
@@ -0,0 +1,24 @@
+#Class 6
+##Facebook Design Activity
+You are designing Facebook's REST API. Come up with a set of routes to do everything Facebook does. When done, read and compare Facebook's API documentation to your mock API, and have a discussion in your group about what is different and what is the same?
```

Comment:
Facebook has (atleast) 3 distinct APIs, I might specify one specifically and provide a link to the documentation (https://developers.facebook.com/docs/graph-api/reference, perhaps). So folks know what they should be comparing their own design against.